### PR TITLE
Electra spec config and builder

### PR DESCRIPTION
**Pull request by rolfyone**
There were fields missing for EIP-7251, and also some of the attestation fields. I added the entire diff from consensus-specs while I was going, it seemed easiest...

## Documentation

- [X] I thought about documentation and added the `doc-change-required` label to this PR if updates are required.

## Changelog

- [X] I thought about adding a changelog entry, and added one if I deemed necessary.

### CODE REVIEW

**Review by courtneyeh — APPROVED** [review]
LGTM

File: ethereum/spec/src/main/java/tech/pegasys/teku/spec/config/SpecConfigElectra.java [hunk 1]
```diff
@@ -31,6 +31,30 @@ static SpecConfigElectra required(final SpecConfig specConfig) {
...
+
+  int getMaxAttesterSlashingsElectra();
+
+  int getMaxAttestationsElectra();
+
+  int getMaxConsolidations();
+
+  int getMaxPartialWithdrawalsPerPayload();
+
+  UInt64 getMinPerEpochChurnLimitElectra();
```
Comment (lucassaldanha) [inline ethereum/spec/src/main/java/tech/pegasys/teku/spec/config/SpecConfigElectra.java:56]: Are we missing `MAX_PER_EPOCH_ACTIVATION_EXIT_CHURN_LIMIT`?

https://github.com/ethereum/consensus-specs/blob/dev/configs/mainnet.yaml#L168C1-L168C42

File: ethereum/spec/src/main/resources/tech/pegasys/teku/spec/config/configs/mainnet.yaml [hunk 2]
```diff
@@ -141,4 +141,7 @@ MAX_REQUEST_BLOB_SIDECARS: 768
 # `2**12` (= 4096 epochs, ~18 days)
 MIN_EPOCHS_FOR_BLOB_SIDECARS_REQUESTS: 4096
 # `6`
-BLOB_SIDECAR_SUBNET_COUNT: 6
\ No newline at end of file
+BLOB_SIDECAR_SUBNET_COUNT: 6
+
+# [New in Electra:EIP7251]
+MIN_PER_EPOCH_CHURN_LIMIT_ELECTRA: 128000000000 # 2**7 * 10**9 (= 128,000,000,000)
```
Comment (lucassaldanha) [inline ethereum/spec/src/main/resources/tech/pegasys/teku/spec/config/configs/mainnet.yaml]: Same as above, are we missing `MAX_PER_EPOCH_ACTIVATION_EXIT_CHURN_LIMIT`? (also applies to minimal.yaml)

File: ethereum/spec/src/main/java/tech/pegasys/teku/spec/config/SpecConfigElectra.java (hunk 1 above)
Comment (rolfyone) [reply ethereum/spec/src/main/java/tech/pegasys/teku/spec/config/SpecConfigElectra.java:56]: it does look like thats needed, will add it! good catch.

**Pull request merged**